frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -81,9 +81,13 @@ function OverViewTab(): JSX.Element {
             ? [
                   {
                       title: 'Tags',
-                      dataIndex: 'tags' as keyof FeatureFlagType,
-                      render: function Render(tags: FeatureFlagType['tags']) {
-                          return tags ? <ObjectTags tags={tags} staticOnly /> : null
+                      dataIndex: 'tags',
+                      render: function Render(_, featureFlag: FeatureFlagType) {
+                          const isExperiment = (featureFlag.experiment_set || []).length > 0
+                          const tags = isExperiment
+                              ? [...(featureFlag.tags || []), 'experiment']
+                              : [...featureFlag.tags]
+                          return featureFlag.tags ? <ObjectTags tags={tags} staticOnly /> : null
```

Comment:
We should only add it to the flags table for now because the tags are always static here and if we add it to the feature flag itself too, it'll have unintended behavior unless we trigger an API create call for it (which seems unnecessary for now)

Reply:
Another option: You can handle this seamlessly by adding tags on the backend instead, since this is the default tag to be added for experiments, and then you don't have to worry about handling it anywhere in the UI (feature flags page, the table, any search whenever implemented)

Right here -> https://github.com/PostHog/posthog/blob/master/ee/clickhouse/views/experiments.py#L100

Reply:
yeah but what about for our existing experiment feature flags :o

Reply:
since the serializer already has the taggedItem mixin, it should handle things pretty easily, you just need to add a tags array in there

Reply:
new migration 😅